src/App.scss

```diff
@@ -7,3 +7,70 @@
   min-height: 100vh;
   font-family: 'Rubik', sans-serif;
 }
+
+.UpdateSWContainer {
```

Comment:
Please check using use onstead of import https://css-tricks.com/introducing-sass-modules/